FlightReservation/src/screens/LogIn.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable prettier/prettier */
+import React, {useState} from 'react';
+import {View, Text, Button, SafeAreaView} from 'react-native';
+import {Provider} from 'react-redux';
+import {store} from '../store/store';
+import {Formik} from 'formik';
+import TextField from '../components/form/TextField';
+import PasswordField from '../components/form/PasswordField';
+import {signUpValidationSchema} from '../schemas/signUpSchema';
+import {signInWithNameEmailAndPassword} from '../helpers/firebaseSignUp';
+
+const LogIn = function () {
+  const [loading, setLoading] = useState(false);
+//   const [emailInUseError, setEmailInUseError] = useState(false);
+
+  const handleSignIn = values => {
+    const {email, password} = values;
+    setLoading(true);
+    signInWithNameEmailAndPassword(email, password)
+      .then(() => setEmailInUseError(false))
+      .catch(() => setEmailInUseError(true))
+      .finally(() => setLoading(false));
+  };
```

Comment:
If you can try to pass this function to a separate file, for example Validations.js, so as not to overload this component.